test-project/src/test_contract.rs

```diff
@@ -53,8 +53,8 @@ impl Instance {
             data
         };
         let code_hash = [
-            41, 2, 122, 125, 224, 108, 110, 209, 243, 246, 117, 145, 214, 14, 89, 13, 162, 122,
-            197, 124, 57, 21, 90, 127, 101, 94, 152, 68, 215, 159, 216, 127,
+            175, 190, 2, 39, 246, 101, 99, 93, 122, 83, 182, 185, 58, 200, 206, 0, 165, 35, 80,
```

Comment:
Why did we have to update code hashes if nothing about the contracts changed?

Reply:
I don't know, I'm not an ink compiler :)

Reply:
Fair, I can accept that explanation.